### PR TITLE
Removed usage of Travis

### PR DESCRIPTION
**Pull request by mdjnelson**

### CODE REVIEW

**Comment by mattporritt** [comment]
Now that the github actions are merged in and setup. 
Can you add the GHA status banner to the readme instead of the travis one?

**Comment by mdjnelson** [comment]
Just saw this comment (went to personal email, doh). I have made that request (i think). This is the first time I have added a GHA status file to a README.md file.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/catalyst/moodle-tool_broadcast.svg?branch=master)](https://travis-ci.org/catalyst/moodle-tool_broadcast)
+![master](https://github.com/catalyst/moodle-tool_broadcast/actions/workflows/moodle-ci.yml/badge.svg)
```
Comment (dmitriim) [inline README.md:1]: you probably need something like Build Status

Comment (mdjnelson) [reply README.md:1]: The `ci.yml` file is called `moodle-ci.yml`. I just followed https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name. I will go with your suggestion but amend the file name.

Comment (mdjnelson) [reply README.md]: It ends up being the same actually afaict.

Comment (mdjnelson) [reply README.md]: On https://github.com/catalyst/moodle-tool_broadcast/tree/removetravis anyways.

**Pull request merged**